### PR TITLE
fix: add apisix/plugin/limit-req to makefile

### PR DESCRIPTION
**Pull request by zll600**
### Description

This pr want to fix [ci problem](https://github.com/apache/apisix/actions/runs/8015080220/job/21895612978?pr=10887) introduced by https://github.com/apache/apisix/pull/10874

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

File: ci/common.sh [hunk 1]
```diff
@@ -170,7 +170,7 @@ GRPC_SERVER_EXAMPLE_VER=20210819
 
 linux_get_dependencies () {
     apt update
-    apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libpcre3 libpcre3-dev libldap2-dev
+    apt install -y cpanminus build-essential libncurses5-dev libreadline-dev libssl-dev perl libpcre3 libpcre3-dev libldap2-dev gawk
```
Comment (shreemaan-abhishek) [inline ci/common.sh:173]: is this needed?

Comment (zll600) [reply ci/common.sh:173]: when I test this pr locally, this is needed.

**Review by membphis — DISMISSED** [review]

File: ci/common.sh (hunk 1 above)
Comment (shreemaan-abhishek) [reply ci/common.sh:173]: that doesn't explain this change 🤔

Comment (zll600) [reply ci/common.sh:173]: I do this according to the Makefile
https://github.com/apache/apisix/blob/869d095153df75fbe6e3edcf631340965fdd04ab/Makefile#L117-L123

Comment (monkeyDluffy6017) [reply ci/common.sh:173]: please revert this change

**Comment by zll600** [comment]
@monkeyDluffy6017 @shreemaan-abhishek PTAL

**Review by shreemaan-abhishek — APPROVED** [review]

**Pull request merged**